src/protagonist/Portal/Pages/Error.cshtml.cs

```diff
@@ -1,27 +1,40 @@
-using System.Diagnostics;
+﻿using System.Net;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using Microsoft.Extensions.Logging;
+using Microsoft.AspNetCore.WebUtilities;
 
 namespace Portal.Pages;
 
-[ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
-[IgnoreAntiforgeryToken]
 public class ErrorModel : PageModel
 {
-    public string RequestId { get; set; }
+    public HttpStatusCode Code { get; set; }
 
-    public bool ShowRequestId => !string.IsNullOrEmpty(RequestId);
-
-    private readonly ILogger<ErrorModel> _logger;
+    public string? Message { get; set; }
+        
+    public IActionResult OnGet(HttpStatusCode code)
+    {
+        // Return Bad Request if an invalid status code is provided
+        if (code == 0)

```

Comment:
I haven't tested this, but would something like `code.Equals(default(HttpStatusCode))` work? Avoids having a magic number

Reply:
It does! I'll patch that in, thanks.